TidepoolOnboarding/View Models/OnboardingViewModel.swift

```diff
@@ -47,7 +47,7 @@ class OnboardingViewModel: ObservableObject, CGMManagerOnboarding, PumpManagerOn
     @Published var cgmManagerIdentifier: String?
     @Published var pumpManagerIdentifier: String? {
         didSet {
-            self.pumpSupportedIncrements = nil
+            self._pumpSupportedIncrements = nil
```

Comment:
Okay, that is **very** annoying. I hate this about Swift protocols. That is, the huge potential for namespace collisions with zero recourse.